### ISSUE TITLE
Where are the old pkg?

### ISSUE THREAD

**Issue by orefalo** [jorgebucaran/fisher#501]
any idea where these pkg are?

fisherman/git_util -> jorgebucaran/fish-git-util
fisherman/debug
fisherman/cprintf -> Markcial/cprintf
fisherman/pwd_is_home -> jorgebucaran/FISH-pwd-is-home
fisherman/last_job_id -> jorgebucaran/FISH-last-job-id
fisherman/humanize_duration -> jorgebucaran/FISH-humanize_duration

plz add a search function to your pkg manager.

**Comment by jorgebucaran** [comment]
@orefalo Some have been destroyed because no one claimed or because they were useless, others transferred to their respective owners and others currently being rewritten. The FISH prefix is just a personal reminder that I need to rewrite them (mostly documentation at this point).

https://github.com/jorgebucaran/fisher/issues/446

> plz add a search function to your pkg manager.

Open a separate issue with your proposal and let's discuss it.

**Comment by jorgebucaran** [comment]
@orefalo Some have been destroyed, transferred to their respective owners or being rewritten. The FISH prefix is just a personal reminder that I need to rewrite those (mostly documentation at this point).

See: #446

> plz add a search function to your pkg manager.

Open a separate issue with your proposal and let's discuss it.

**Comment by jorgebucaran** [comment]
@orefalo Some have been destroyed, transferred to their respective owners or being rewritten. In the ones you mention, only **fisherman/debug** perished after #446. The FISH prefix is just a personal reminder that I need to rewrite those (mostly documentation at this point).

> plz add a search function to your pkg manager.

Open a separate issue with your proposal and let's discuss it.

**Comment by jorgebucaran** [comment]
@orefalo Some were destroyed, transferred to their respective owners or are being rewritten. From your list, only **fisherman/debug** perished. See #446. The FISH prefix is just a personal reminder that I need to rewrite those (mostly documentation at this point).

> plz add a search function to your pkg manager.

Open a separate issue with your proposal and let's discuss it.
